sunpy/instr/fermi.py

```diff
@@ -41,25 +42,25 @@ def download_weekly_pointing_file(date):
     # use Fermi data server to access weekly LAT pointing file.
     base_url = 'ftp://legacy.gsfc.nasa.gov/FTP/glast/data/lat/weekly/spacecraft/'
     fbasename = 'lat_spacecraft_weekly_w'
-    
+
     # find out which file to get based on date
     # earliest full file in the FERMI server is for mission week 10,
     # beginning 2008 August 7.
     weekly_file_start = parse_time('2008-08-07')
     base_week = 10
 
     # find out which mission week corresponds to date
-    time_diff = date-weekly_file_start
-    weekdiff = time_diff.days//7
+    time_diff = date - weekly_file_start
+    weekdiff = time_diff.days // 7
```

Comment:
wait... cannot we get directly the week number from the date and substract these?